scenarios/ocd/CreateWordPressDeployment/README.md

```diff
@@ -0,0 +1,466 @@
+# Quickstart: Deploy a Scalable & Secure WordPress instance on AKS
+
+Welcome to this tutorial where we will take you step by step in creating an Azure Kubernetes Web Application that is secured via https. This tutorial assumes you are logged into Azure CLI already and have selected a subscription to use with the CLI. It also assumes that you have Helm installed ([Instructions can be found here](https://helm.sh/docs/intro/install/)).
+
+## Define Environment Variables
+
+The First step in this tutorial is to define environment variables.
+
+```bash
+export SSL_EMAIL_ADDRESS="$(az account show --query user.name --output tsv)"
+export NETWORK_PREFIX="$(($RANDOM % 253 + 1))"
+export RANDOM_ID="$(openssl rand -hex 3)"
+export MY_RESOURCE_GROUP_NAME="myResourceGroup$RANDOM_ID"
```

Comment:
The resource group name should be more specific. Something like 'myWordPressAKSResourceGroup$RANDOM_ID'